app/models/package.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Package < ApplicationRecord
+  enum status: {
+    pending: 0,
+    paid: 1,
+  }
```

Comment:
I think the `frequency` attribute on the packages table should be the enumerable not `status`. I also think we should set the default value to the first option, see docs [here](https://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html) for more info. Thoughts?

```suggestion
  enum frequency: {
    daily: 0,
    weekly: 1,
    biweekly: 2,
    monthly: 3,
    yearly: 4,
  }, default: :daily
```